src/styles/card.scss

```
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+
+    &__name {
+      color: #34568b;
+    }
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+    margin-bottom: 16px;
+    color: $grey-color;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+
+  &__review {
+    display: flex;
+    width: 166px;
+    height: 16px;
+    margin-bottom: 24px;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__review-text {
+    margin-top: 12px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__button {
+    font-size: 14px;
+    text-align: center;
+    font-weight: 700;
+    line-height: 16px;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    background: $blue-color;
+    color: $white-color;
+    border: 1px solid $white-color;
+    transition:
+      background-color 300ms,
+      color 300ms;
+
+    &:hover {
+      color: $blue-color;
+      background-color: $white-color;
+      border: 1px solid $blue-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    margin-top: 0;
+    margin-bottom: 0;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $grey-color;
+  }
+
+  &__price-number {
+    margin-top: 0;
+    margin-bottom: 0;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
```

Use scss nesting, fix it everywhere
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    margin-bottom: 16px;
    
    &-text {
      // write styles here
    }
    
    &-number {
      // write styles here
    }
  }
```